spec/index.html

```diff
@@ -1284,8 +1284,65 @@ <h2>Best Practices</h2>
     <section id="extended-profile" class="informative">
       <h2>Extended YAML-LD Profile</h2>
 
+      <section>
+        <h2>Motivation</h2>
+
+        <p>
+          The YAML-LD specification relies upon YAML to serialize Linked Data to the extent
+          that YAML is compatible with JSON, which simplifies the operation and usage
+          of YAML-LD. However, the <a href="#json-vs-yaml">more
+          expressive feature set</a> of YAML invites us to represent
+          Linked Data in a more expressive way.
+        </p>
+
+        <section>
+          <h2>Specify node <code>@type</code></h2>
+
+          <p>In RDF, <code>@type</code> translates to one of the following:</p>
+
+          <ul>
+            <li>an <code>rdf:type</code> edge</li>
+            <li>a <code>datatype</code> mark for a Literal node</li>
+          </ul>
+
+          <p>Possible ways to specify this in YAML-LD are the following:</p>
+
+          <ul>
+            <li>In the <code>@context</code>, only to say that the node is an IRI</li>
+            <li>Inline, using the <code>@type</code> keyword</li>
+            <li>Using a YAML Tag, as shown below</li>
+          </ul>
+
+          <pre
+            class="example yaml"
+            data-transform="updateExample"
+            data-result-for="YAML-LD: Tags"
+            data-content-type="application/ld+json"
+            title="YAML-LD with tags"
+          >
+            %TAG !xsd: http://www.w3.org/2001/XMLSchema%23
+            ---
+            "@id": yaml-ld
+            updated-at: !xsd:date 2023-03-21
+          </pre>
+        </section>
+
+        <p>
+          Here, <code>%TAG</code> declares the <code>!xsd:</code> prefix for tags used
+          in the document. YAML treats tags as IRIs, which brings it close to the LD family
+          of data formatting tools. Note that the directives section must be separated
+          from the main document with <code>---</code> (a line containing only three hyphens).
+        </p>
```

Comment:
As I read this, the `<pre ... />` segment is only applicable to the final list item, and the subsequent paragraph is specific to the `<pre>`, so these should be embedded in the `<li>`, as --
```suggestion
              <li>
                <p>Using a YAML Tag, as shown below:</p>
                <pre
                  class="example yaml"
                  data-transform="updateExample"
                  data-result-for="YAML-LD: Tags"
                  data-content-type="application/ld+json"
                  title="YAML-LD with tags"
                >
                %TAG !xsd: http://www.w3.org/2001/XMLSchema%23
                ---
                "@id": yaml-ld
                updated-at: !xsd:date 2023-03-21
              </pre>
              <p>
                Here, <code>%TAG</code> declares the <code>!xsd:</code> prefix for tags used
                in the document. YAML treats tags as IRIs, which brings it close to the LD family
                of data formatting tools. Note that the directives section must be separated
                from the main document with <code>---</code> (a line containing only three hyphens).
              </p>
            </li>
          </ul>
        
        </section>     
```

Reply:
Done that :ok:

Reply:
Incompletely done... It would have been better `committed` from this suggestion, than re-worked in your own commit.

Note that I also moved the `<p>` which is now between `</section>` and `<section>`, also into the `<li>`.